src/hammer-vlsi/utils_test.py

```diff
@@ -49,6 +51,44 @@ def test_optional_map(self) -> None:
         self.assertNotEqual(optional_map("88", str_to_num), "880")
         self.assertEqual(optional_map("42", str_to_num), 420)
 
+    def test_coerce_to_grid(self) -> None:
+        self.assertEqual(coerce_to_grid(1.23, Decimal("0.1")), Decimal("1.2"))
+        self.assertEqual(coerce_to_grid(1.23, Decimal("0.01")), Decimal("1.23"))
+        self.assertEqual(coerce_to_grid(1.227, Decimal("0.01")), Decimal("1.23"))
```

Comment:
Can we also check something like `coerce_to_grid(1/3, Decimal("0.001"))`

Reply:
Sure, will add.